lib/cc/engine/analyzers/ruby/main.rb

```diff
@@ -33,8 +34,47 @@ def overage(issue)
           end
 
           def process_file(file)
-            RubyParser.new.process(File.binread(file), file, TIMEOUT)
+            if File.extname(file) == ".erb"
```

Comment:
I actually prefer the single file because I think the `main` class is fairly small (< 5 methods), and adding encapsulating parser files might be adding needless encapsulation.

Let me know if you feel strongly though - and I can make a change.